packages/client/package.json

```diff
@@ -69,6 +70,7 @@
     "prettier": "^1.19.1",
     "react-hook-form": "^4.9.6",
     "react-test-renderer": "^16.12.0",
+    "serverless": "^1.64.1",
```

Comment:
Is it possible to move `serverless`  dependency to the  `<root>/package.json` and using lerna share across client and services? It's currently used in two `package.json` files and what's took my attention, we already are using two different versions of the same package. It won't hurt us.. until it hurts us 🤷‍♂️

https://github.com/BuildIt-Poland/buddy-app/blob/4d6235f9f01cb7c257e44155e14127655d1f8708/packages/services/package.json#L38

https://github.com/BuildIt-Poland/buddy-app/blob/bb1cfb48681c6f0706629711ceea16b52f7eaec5/packages/client/package.json#L73